spec/actions_manager_spec.coffee

```diff
@@ -92,10 +92,13 @@ action_reporting_tests = ->
       expect(@payload.actions[0]).to.be.an('object')
 
     it 'has proper data in the action\'s object', ->
+      data = JSON.parse(@data)
+      data.rpos = reported_position if reported_position != undefined
+
       expect(@payload.actions[0]).to.eql
         category: @category
         type: @type
-        data: @data
+        data: JSON.stringify(data)
```

Comment:
do we need `JSON.stringify` in here?

Reply:
Yes because in the tests we pass always data as string